src/wgquickprocess.cpp

```diff
@@ -37,18 +38,18 @@ QString scriptPath() {
 }  // namespace
 
 // static
-bool WgQuickProcess::createConfigFile(
-    const QString& configFile, const QString& privateKey,
-    const QString& deviceIpv4Address, const QString& deviceIpv6Address,
-    const QString& serverIpv4Gateway, const QString& serverIpv6Gateway,
-    const QString& serverPublicKey, const QString& serverIpv4AddrIn,
-    const QString& serverIpv6AddrIn, const QString& allowedIPAddressRanges,
-    int serverPort, bool ipv6Enabled) {
-  Q_UNUSED(serverIpv6AddrIn);
+bool WgQuickProcess::validateWgArgs(
+    const QString& privateKey, const QString& deviceIpv4Address,
+    const QString& deviceIpv6Address, const QString& serverIpv4Gateway,
```

Comment:
This change would break windows build. On windows, we call this: https://github.com/mozilla-mobile/mozilla-vpn-client/blob/main/src/platforms/windows/daemon/windowsdaemon.cpp#L418
...passing the file name. The file is overwritten each time.